stock_release_channel/models/stock_preferred_release_channel.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2024 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import api, fields, models
+
+
+class StockPreferredReleaseChannel(models.Model):
+    _name = "stock.preferred.release.channel"
```

Comment:
@sebalix @jbaudoux Why not having put this in a separate 'stock_release_channel_preferred_channel' module ? I'd like introducing new features separately to:

- lower the code
- identify a functionality more clearly

Reply:
@rousseldenis @jbaudoux code moved into two new modules:

- `stock_release_channel_partner_by_date`
- `sale_stock_release_channel_partner_by_date`